src/frontend/next/src/components/SearchBar.tsx

```diff
@@ -80,9 +80,9 @@ const useStyles = makeStyles((theme: Theme) =>
 );
 
 type searchBarProps = {
-  text: string;
+  text: string | null;
```

Comment:
All these null types: can we not assume that text/string values are either a string of `''` (empty string) so nothing breaks, and it will simplify a bunch of code later?